mod/profiles/set/self/metric_answer.rb

```diff
@@ -38,3 +38,7 @@ def created_query_variant user_id, variant=nil
     {}
   end
 end
+
+def contribution_report_title
+  "Metric values"
+end
```

Comment:
!

Reply:
Confused.  We want the title to be "Answers", right?

(Though this does make the point that we don't have to change the cardname for that).

Reply:
I don't know if we want a change in the messaging. This PR never changed anything that is visble by users (to my knowledge). Your very first comment gave the impression that didn't notice that so I highlighted this.

Reply:
Now i understand.  You're right, I missed that. I confirmed that we do want "answers"